src/java/com/sarality/app/view/list/ListComponentSorter.java

```diff
@@ -0,0 +1,14 @@
+package com.sarality.app.view.list;
+
+import java.util.List;
+
+/**
+ * Interface that would sort the list in the order that the listview requires.
+ *
+ * @param <T> Type of data
+ * @author sunayna@ (Sunayna Uberoy)
+ */
+public interface ListComponentSorter<T> {
```

Comment:
We can use Collections.sort(List, Comparator) directly. Use can pass in a Comparator to the ListComponent as the sorter.